### PR TITLE
frontend/feature/#1840/driver-Onboarding-Flow-Revamped

### PR DESCRIPTION
**Pull request by mohdshoaib18169**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

File: Frontend/android-native/app/src/dev/assets/juspay/becknbase.html [hunk 1]
```diff
@@ -34,7 +34,7 @@
     var ip = DUIGatekeeper.getResourceByName("local_ip");
     var port = DUIGatekeeper.getResourceByName("dev_port");
     port = port ? port : "8080";
-    var url = "http://" + ip + ":" + port;
+    var url = "http://" + "192.168.10.10" + ":" + port;
```
Comment (Anuragini-2506) [inline Frontend/android-native/app/src/dev/assets/juspay/becknbase.html:37]: please remove the ip address from becknbase.html

File: Frontend/android-native/app/src/driver/jatriSaathiDriver/res/drawable/ny_ic_welcome_screen_2.png
Comment (Anuragini-2506) [inline Frontend/android-native/app/src/driver/jatriSaathiDriver/res/drawable/ny_ic_welcome_screen_2.png:1]: please compress the image ( try below 15KB)

File: Frontend/ui-common/src/Engineering/Helpers/JBridge.js [hunk 2]
```diff
@@ -1273,6 +1273,7 @@ export const renderBase64Image = function (image) {
     return function (fitCenter) {
       try {
         if (JBridge.renderBase64Image) {
+          console.log("my name");
```
Comment (Anuragini-2506) [inline Frontend/ui-common/src/Engineering/Helpers/JBridge.js:1276]: please remove the log statements if not required

File: Frontend/ui-common/src/Engineering/Helpers/JBridge.js [hunk 3]
```diff
@@ -1616,4 +1617,33 @@ export const launchDateSettings = function (res) {
   if(JBridge.launchDateSettings){
     return JBridge.launchDateSettings();
   }
+};
+
+export const onFocused = function(callback){
+   // console.log ("See You");
+  //return function(elementId) {
+    return function(elementId) {
```
Comment (Anuragini-2506) [inline Frontend/ui-common/src/Engineering/Helpers/JBridge.js:1625]: try to remove commented code if it is not required

File: Frontend/ui-driver/src/Screens/AddVehicleDetailsScreen/Controller.purs [hunk 4]
```diff
@@ -172,12 +229,15 @@ eval (CallBackImageUpload base_64 imageName imagePath) state = do
...
   if base_64 /= "" then do
-    let newState = state { props = state.props { rcAvailable = true, rc_name = imageName, isValidState = (checkRegNum (state.data.vehicle_registration_number))}, data = state.data { rc_base64 = base_64 }}
+    let newState = state { props = state.props { validateProfilePicturePopUp = true, imageCaptureLayoutView = false, rcAvailable = true, rc_name = imageName, isValidState = (checkRegNum (state.data.vehicle_registration_number))}, data = state.data { rc_base64 = base_64 }}
     exit $ ValidateImageAPICall newState
     else continue state{props{isValidState = false}}
-eval (UploadFile) state = continueWithCmd state [do
-        _ <- liftEffect $ uploadFile unit
-        pure NoAction]
+eval (UploadFile) state = continueWithCmd (state {props {validateProfilePicturePopUp = false, imageCaptureLayoutView = true}}) [do
+    --  _ <- pure $ toast("Hello")
```
Comment (Anuragini-2506) [inline Frontend/ui-driver/src/Screens/AddVehicleDetailsScreen/Controller.purs:236]: please remove the commented code if not required.

File: Frontend/ui-driver/src/Screens/ApplicationStatusScreen/View.purs [hunk 5]
```diff
@@ -137,17 +129,79 @@ applicationStatus state push =
...
+  , height MATCH_PARENT
+  ][  linearLayout
+            [ width MATCH_PARENT
+            , height MATCH_PARENT
+            , margin $ MarginBottom 20
+            ][
+               textView
+              [   width WRAP_CONTENT
+               , height WRAP_CONTENT
+               , text ("Start earning in 3 simple step")--"Start earning in 3 simple step"
```
Comment (Anuragini-2506) [inline Frontend/ui-driver/src/Screens/ApplicationStatusScreen/View.purs:142]: try to get the text via getString method and define its translations accordingly in Strings folder.

File: Frontend/ui-driver/src/Screens/ApplicationStatusScreen/View.purs [hunk 6]
```diff
@@ -137,17 +129,79 @@ applicationStatus state push =
...
+               , textSize FontSize.a_14
+               , fontStyle $ FontStyle.medium LanguageStyle
+              ]
+              ,  textView
+              [  width MATCH_PARENT
+               , height WRAP_CONTENT
+               , gravity RIGHT
+               , text (show (completePercentage state) <> " % " <> ("Complete"))
```
Comment (Anuragini-2506) [inline Frontend/ui-driver/src/Screens/ApplicationStatusScreen/View.purs:150]: try to get the text via getString method and define its translations accordingly in Strings folder.

File: Frontend/ui-driver/src/Screens/PermissionsScreen/View.purs [hunk 7]
```diff
@@ -156,12 +177,12 @@ checkBox item state =
...
         , width (V 18)
         , stroke ("1," <> Color.black)
-        , cornerRadius 2.0
+        , cornerRadius 9.0
         ][]
         , imageView
         [ width (V 18)
         , height (V 18)
-        , imageWithFallback "ny_ic_check_box,https://assets.juspay.in/nammayatri/images/driver/ny_ic_check_box.png"
+        , imageWithFallback "ny_ic_check_mark,https://assets.juspay.in/nammayatri/images/driver/ny_ic_check_mark.png"
```
Comment (Anuragini-2506) [inline Frontend/ui-driver/src/Screens/PermissionsScreen/View.purs:185]: can you try to use getAssetStoreLink ?

File: Frontend/ui-driver/src/Screens/ReportIssueChatScreen/Controller.purs [hunk 8]
```diff
@@ -252,7 +252,7 @@ eval (AddImagesModelAction (ImageModel.OnClickDelete index)) state = do
   let imageIds' = fromMaybe state.data.addImagesState.imageMediaIds $ deleteAt index state.data.addImagesState.imageMediaIds
   continueWithCmd state { data { addImagesState { images = images', stateChanged = not (imageIds' == state.data.uploadedImagesIds), imageMediaIds = imageIds' } } } [do
     _ <- forWithIndex images' \i x -> do
-      _ <- renderBase64ImageFile x.image (getNewIDWithTag "add_image_component_image" <> (show i)) false
+      _ <- renderBase64ImageFile x.image (getNewIDWithTag "add_image_component_image" <> (show i)) false false
```
Comment (Anuragini-2506) [inline Frontend/ui-driver/src/Screens/ReportIssueChatScreen/Controller.purs:255]: use  PascalCase for ids

File: Frontend/ui-driver/src/Strings/TA.js [hunk 9]
```diff
@@ -492,5 +492,6 @@ const tamilStrings = {
   YOU : " (நீங்கள்)",
   DAILY : "தினசரி",
   WEEKLY : "வாரந்தோறும்",
-  ACCEPT_RIDES_TO_ENTER_RANKINGS : "தரவரிசையில் நுழைய சவாரிகளை ஏற்கவும்!"
+  ACCEPT_RIDES_TO_ENTER_RANKINGS : "தரவரிசையில் நுழைய சவாரிகளை ஏற்கவும்!",
+  WELCOME_TEXT_LANGUAGE_SCREEN : ""
```
Comment (Anuragini-2506) [inline Frontend/ui-driver/src/Strings/TA.js:496]: why the WELCOM_TEXT_LANGUAGE_SCREEN is an empty string ?

**Comment by witcher-shailesh** [comment]
Not required. cc: @ashkriti

**Pull request closed without merging**